t/mojolicious/perl_config_lite_app.t

```diff
@@ -0,0 +1,21 @@
+use Mojo::Base -strict;
+
+BEGIN {
+  $ENV{MOJO_MODE}    = 'development';
+  $ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll';
+}
+
+use Test::Mojo;
+use Test::More;
+use Mojo::File qw(curfile);
+use Mojolicious::Lite;
+
+plugin 'Config';
+is_deeply app->config, {
+		foo => "bar",
+		utf => "утф",
+		file => curfile->sibling('perl_config_lite_app.conf')->to_string,
+		line => 7,
+	}, 'right value';
```

Comment:
That also does not look like it follows our formatting rules.

Reply:
Fixed as above